### PR TITLE
Smart charging

### PR DESCRIPTION
**Pull request by leongmichael**

### CODE REVIEW

**Comment by leongmichael** [comment]
image
image

File: server/python-backend/scripts/post-process/smart-charging-analysis.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        'average_energy_per_session': df_filtered['energy'].mean(),
+        'total_duration': df_filtered['duration'].sum(),
+        'average_duration_per_session': df_filtered['duration'].mean(),
+        'L2_sessions': len(df_filtered[df_filtered['station_type'] == 'L2']),
+        'L3_sessions': len(df_filtered[df_filtered['station_type'] == 'L3']),
+    }
+    return stats
+
+def main():
+    base_dir = os.getcwd() + "\\python-backend\\tests"
```
Comment (rongxinyin) [inline server/python-backend/scripts/post-process/smart-charging-analysis.py]: base dir path should work for all os systems.

**Review by rongxinyin — CHANGES_REQUESTED** [review]
testing data path that should work for different os systems.

File: server/python-backend/scripts/post-process/smart-charging-analysis.py (hunk 1 above)
Comment (rongxinyin) [reply server/python-backend/scripts/post-process/smart-charging-analysis.py]: fix the base dir using a relative path.

**Pull request merged**